frontend/src/scenes/authentication/InviteSignup.tsx

```diff
@@ -221,49 +232,73 @@ function UnauthenticatedAcceptInvite({ invite }: { invite: PrevalidatedInvite })
-                        placeholder="••••••••••"
-                        autoComplete="new-password"
-                        autoFocus={window.screen.width >= 768} // do not autofocus on small-width screens
-                        disabled={isSignupSubmitting}
-                    />
-                </LemonField>
+                {!areExtraFieldsHidden && (
+                    <>
+                        <LemonField
+                            name="password"
+                            label={
+                                <div className="flex flex-1 items-center justify-between">
+                                    <span>Password</span>
+                                    <PasswordStrength validatedPassword={validatedPassword} />
+                                </div>
+                            }
+                        >
+                            <LemonInput
+                                type="password"
+                                className="ph-ignore-input"
+                                data-attr="password"
+                                placeholder="••••••••••"
+                                autoComplete="new-password"
+                                autoFocus={window.screen.width >= 768} // do not autofocus on small-width screens
+                                disabled={isSignupSubmitting}
+                            />
+                        </LemonField>
 
-                <LemonField
-                    name="first_name"
-                    label="First Name"
-                    help={
-                        invite?.first_name ? 'Your name was provided in the invite, feel free to change it.' : undefined
-                    }
-                >
-                    <LemonInput data-attr="first_name" placeholder="Jane" />
-                </LemonField>
+                        <LemonField
+                            name="first_name"
+                            label="First Name"
+                            help={
+                                invite?.first_name
+                                    ? 'Your name was provided in the invite, feel free to change it.'
+                                    : undefined
+                            }
+                        >
+                            <LemonInput data-attr="first_name" placeholder="Jane" />
+                        </LemonField>
 
-                <SignupRoleSelect />
+                        <SignupRoleSelect />
+                    </>
+                )}
 
-                <LemonButton
-                    type="primary"
-                    status="alt"
-                    htmlType="submit"
-                    data-attr="password-signup"
-                    loading={isSignupSubmitting}
-                    center
-                    fullWidth
-                >
-                    Continue
-                </LemonButton>
+                {precheckResponse.status === 'pending' || !precheckResponse.sso_enforcement ? (
```

Comment:
Are these conditionals correct? If the response status is pending do we want to show the "continue" button?

Reply:
Will test a few things and confirm / improve.